starklings-backend/starklings_backend/models.py

```diff
@@ -0,0 +1,43 @@
+from sqlalchemy import create_engine, Column, Integer, String, ForeignKey
+from sqlalchemy.orm import sessionmaker, relationship, declarative_base
+# from db import Base
+
+Base = declarative_base()
+
+class StarklingsUser(Base):
+    __tablename__ = "starklings_user"
+    wallet_address = Column(String(42), primary_key=True)
+    signature = Column(String(255), nullable=False)
+    github = Column(String(255), nullable=False)
+    username = Column(String(255), nullable=False)
+    score = Column(Integer, nullable=False, default=0)
+    starklings_user = relationship("ValidatedExercise")
```

Comment:
rename it to validated_exercises

Reply:
according to the [docs](https://docs.sqlalchemy.org/en/14/orm/basic_relationships.html), the arg to `relationship` is the `Class` name cmiiw

Reply:
if i change to table name, i'll get this error instead

```
sqlalchemy.exc.ArgumentError: relationship 'starklings_user' expects a class or a mapper argument (received: <class 'sqlalchemy.sql.schema.Table'>)
```